content/docs/2.0/scalers/metrics-api.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Metrics API"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on a metric value from an API"
```

Comment:
This feels more natural to me but that's personal pref, and a big nit😂
```suggestion
description = "Scale applications based on a metric from an API"
```

But not a must though!